tests/kokkos/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# TRACCC library, part of the ACTS project (R&D line)
+#
+# (c) 2022 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+traccc_add_test( kokkos
+   kokkos_main.cpp
```

Comment:
Can we put this into a separate shared/static library like we do for CUDA?

Reply:
:confused: I don't see much point in doing that. Once we want to build multiple separate executables that test Kokkos, we can move to such a setup. But as long as we only have a single executable, it's an unnecessary complication. (And I imagine that we'll only ever have a single executable that would contain all the Kokkos GoogleTests.)

Reply:
OK. :+1: